### DISCUSSION TITLE
Any updates on providing proper vscode github sponsors integration API beyond sponsor link config?

### DISCUSSION THREAD

**Discussion by RandomFractals** [microsoft/vscode-discussions#98] [Extension Development QnA] [answered]
I am just not sure where we ended that discussion in vscode dev community slack back in April.

Sponsor link config you added is nice, but not quite the full github sponsors api integration like github apps have now.

**Comment by isidorn** [comment, marked as answer]
Thanks for raising this question but currently there are no plans to update the VS Code GitHub sponsor integration. Reason for this is that it would require us to bake some sponsor information in the Session API which we do not want to do for now. Hope that helps
